### ISSUE TITLE
max items per feed

### ISSUE THREAD

**Issue by qutetemp** [newsboat/newsboat#164]
Is there any way I can set up max items per feed ?

**Comment by qutetemp** [comment]
Yes, I know but that applies to every rss feed. I'm looking to apply that setting to a specifc feed only.

**Comment by Minoru** [comment]
Ah, that's not possible at the moment, but will be once #83 is implemented.

**Comment by qutetemp** [comment]
ahh, ok thanks

**Comment by Vinfall** [comment]
I know this is an old issue, sorry to dig this up. But I find the latest documentation (2.33) still a bit misleading on this.

In [`max-items`](https://newsboat.org/releases/2.33/docs/newsboat.html#max-items) section it says:

> You can configure how many articles are kept per feed so that the article backlog doesn’t grow endlessly by configuring [max-items](https://newsboat.org/releases/2.33/docs/newsboat.html#max-items).
> Set the number of articles to maximally keep per feed. If the number is set to 0, then all articles are kept.

I think `per feed` is the misleading part. Maybe you could add one line like `This applies to all feeds. It's currently not possible to set a max-item for a specific feed`. It's not precise enough, but I hope you get the point.

**Comment by Minoru** [comment]
The majority of our settings are like that, and I don't want to add such disclaimers to each of them. I reworded the doc to avoid the problematic "per feed" phrase — what do you think of https://github.com/newsboat/newsboat/pull/2583?

**Comment by Vinfall** [comment]
> The majority of our settings are like that

I think I have to agree... What makes me has the impression is actually [ignore-article](https://newsboat.org/releases/2.33/docs/newsboat.html#ignore-article) but that goes like `ignore-article <feed> <filterexpr>` with a clear `<feed>` parameter so I just happened to misread it somehow.
